### PR TITLE
Fix skipped integration tests

### PR DESCRIPTION
**Pull request by laemtl**

### CODE REVIEW

File: modules/imaging_browser/test/imaging_browserTest.php [hunk 1]
```diff
@@ -657,43 +664,38 @@ function testImagingBrowserSiteDependingOnPermissions()
...
-        $this->clickToLoadNewPage($PSCIDHeader);
 
         $FirstEntry = $this->safeFindElement(
             WebDriverBy::cssSelector(
-                ".table > tbody:nth-child(2) > tr:nth-child(1) > td:nth-child(3)"
+                "#dynamictable tbody > tr:nth-child(1) td:nth-child(3)"
             )
         )->getText();
-        $this->assertStringContainsString("BOL0003", $FirstEntry);
+        $this->assertStringContainsString("ROM300", $FirstEntry);
```
Comment (driusan) [inline modules/imaging_browser/test/imaging_browserTest.php:698]: It's hard to tell what the correct value is here from the diff or if the change in behaviour was intentional from a PR when the test was skipped or a bug introduced at some point. Can you include a comment in the code explaining the logic of why it's "ROM300"?

File: modules/imaging_browser/test/imaging_browserTest.php [hunk 2]
```diff
@@ -646,7 +643,17 @@ function testImagingBrowserSiteDependingOnPermissions()
         $SiteFilterText     = $this->safeFindElement(
             WebDriverBy::Name("site")
         )->getText();
-        $this->assertStringContainsString("All", $SiteFilterText);
+        $sites = [
+          'Data Coordinating Center',
```
[the remark this answers is not in the document]
Comment (laemtl) [reply modules/imaging_browser/test/imaging_browserTest.php:647]: Seems like it 
Screen Shot 2023-01-06 at 10 08 24 AM

[the remark this answers is not in the document]
Comment (cmadjar) [reply modules/imaging_browser/test/imaging_browserTest.php:647]: yes we did get rid of the ALL.

File: modules/imaging_browser/test/imaging_browserTest.php (hunk 1 above)
Comment (cmadjar) [reply modules/imaging_browser/test/imaging_browserTest.php:698]: There are no BOL0003 subjects on RB so not sure why it is in the test here. Unless it was testing for subjects that do not exist?

**Comment by laemtl** [comment]
@driusan Rebased

**Pull request merged**